pkgs/tools/package-management/nix-du/default.nix

```diff
@@ -40,5 +40,6 @@ rustPlatform.buildRustPackage rec {
     license = licenses.lgpl3Only;
     maintainers = [ maintainers.symphorien ];
     platforms = platforms.unix;
+    broken = lib.isDarwin;
```

Comment:
Are you sure this is correct? I get `error: attribute 'isDarwin' missing` here.

Reply:
Other places use `stdenv.isDarwin`, not `lib.isDarwin`.

Reply:
Nope, that was a brainfart, should be fixed now. Thanks!